src/table/pagination-table/components/styles.ts

```diff
@@ -27,12 +27,15 @@ export const StyledTableContainer = styled(TableContainer, {
 }));
 
 export const StyledTable = styled(Table, {
-  shouldForwardProp: (prop: string) => prop !== 'customWidth',
-})(({ customWidth }) =>
+  shouldForwardProp: (prop: string) => prop !== 'customWidth' && prop !== 'isBorderRight',
+})(({ customWidth, isBorderRight }) =>
```

Comment:
not sure we want to put the border on the table. It will look slightly different than putting it on the cell. Also, this does not respect if we need the dark mode borders

Reply:
Right, dark mode, Thx.

the border on the table and the cell is no different as far as I can see. And on the table is much simpler.
<img width="337" alt="Screenshot 2023-03-20 at 11 19 43" src="https://user-images.githubusercontent.com/4471489/226311852-20503626-2890-492c-aa70-5a47f89f1e7e.png">
<img width="395" alt="Screenshot 2023-03-20 at 11 00 37" src="https://user-images.githubusercontent.com/4471489/226311901-3a0969e1-8023-4f6c-9a02-18bda8ec82f7.png">

Reply:
there is a small difference, but barely even noticable. If you zoom in on the corners, you see that you get a diagonal intersection kind of. I think we can keep the right border on table level